src/ice_shelf/MOM_ice_shelf.F90

```diff
@@ -1815,6 +1840,8 @@ subroutine initialize_ice_shelf(param_file, ocn_grid, Time, CS, diag, forces_in,
   if (CS%active_shelf_dynamics) then
     CS%id_h_mask = register_diag_field('ice_shelf_model', 'h_mask', CS%diag%axesT1, CS%Time, &
        'ice shelf thickness mask', 'none')
+    CS%id_shelf_sfc_mass_flux = register_diag_field('ice_shelf_model', 'sfc_mass_flux', CS%diag%axesT1, CS%Time, &
+       'ice shelf surface mass flux deposition from atmosphere', 'none')
```

Comment:
I suspect that this register_diag_field call is missing the optional argument `conversion=conversion=US%RZ_T_to_kg_m2s`.

Reply:
Fixed in this [PR](https://github.com/OlgaSergienko/MOM6/pull/10)